mentoring/mentor-guide.md

```diff
@@ -0,0 +1,100 @@
+*  To pair program with you.
+*  Ask you an hours worth of questions (AMA). 
+*  Do code reviews/docs reviews.
+*  Discuss their path so far and ask for input/feedback.
+*  Code base tour of a certain area
+*  Or a mix - it’s up to the mentee with how they’d like to spend the time and your skills/experience.
+* You'll fill out a form once a month to list your availibility.
+
+*[Outreachy](https://www.outreachy.org)*
+* Being a mentor can take anywhere from 2-5 hours a week depending on the scope of the project that the intern(s) work on. 
+* Before the program issue grooming/project planning activities:
+  * For the Outreachy application process that they host [here](https://www.outreachy.org/apply/). All interns that are interested in the project must make one contribution as part of their application. Label them with a good first time issue.
+* During the program:
+  * The time above is split up between slack, 1:1 pair programming and standups.
+  * Provide official feedback during intervals to Outreachy and the intern.
+  * Have read and understand the [rules and guidelines](https://www.outreachy.org/mentor/) around mentoring for Outreachy
+
+CNCF will financially support one Outreachy intern for Kubernetes; if you would like to participate and we already have one, assistance with securing a corporate sponsor for the budget (6.5k) may be needed.
+
+*[Google Summer of Code](google-summer-of-code.md)*
+* Being a mentor can take anywhere from 2-10 hours a week of your time depending on the student. In most cases, you will have a secondary/backup mentor to distribute the workload.
+* Before the Program:
+  * Read the official GSOC mentors manual to get an overview.
+  * Talk to interested candidates via Slack. Share with them the project details, goals and some ideas for implementation.
+  * Point them to self-contained, easy and newcomer-friendly bugs to fix.
+  * Review draft proposals and provide feedback before the application period.
+  * Evaluate the final proposals and select the student you want to mentor.
+* During the program:
+  * Set development and communication plans with your mentee. Make sure you establish realistic work objectives and timeline expectations.
+  * Help them get integrated into the community and encourage them to participate.
+  * Provide guidance such as pointers to useful documentation, code reviews, etc.
+  * Set aside time for a weekly sync via Slack or video call, whichever you prefer.
+  * Collect weekly status reports. They are important in making sure that time-line slippages and scope creep are addressed in a proactive manner.
+  * Keep track of the mentee's progress and keep them informed about their status. Make sure you deliver feedback early.
+  * Be responsive to questions - usual expectation is to reply within 24 hours (occasionally under 36 hours is ok).
+  * There will be three evaluations of your mentee's work. Submit them timely. The evaluation forms take hardly 10-15 min to fill.
```

Comment:
spell out "minutes"

Reply:
Deliberately skipped?  Either way just a suggestion for ease of readability.

Reply:
Oh - I changed another instance and must have missed this one